Modules/Scripted/Home/CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ set(MODULE_NAME Home)
 #-----------------------------------------------------------------------------
 set(ccf_annotation_resolution 25) # Possible values are 25 or 50 (Used in CellLocatorConfig.py)
 
+#-----------------------------------------------------------------------------
+configure_file(
+  ${MODULE_NAME}Lib/CellLocatorConfig.py.in
+  ${CMAKE_CURRENT_BINARY_DIR}/${MODULE_NAME}Lib/CellLocatorConfig.py
+  @ONLY
+  )
+
```

Comment:
I was getting errors about `CellLocatorConfig.py` not found by `slicerMacroBuildScriptedModule`, so I move it here above `set(MODULE_PYTHON_SCRIPTS ...)` so that it is available.

Reply:
Since one of the variables used in the script are set later, let's add this at the top after setting `ccf_annotation_resolution`:

```
set(mni_average_template_filename "mni_icbm152_t1_tal_nlin_sym_09b_hires.nii") # Used in CellLocatorConfig.py
```

Reply:
This was done.